### PR TITLE
Update to new build tools version.

### PR DESCRIPTION
**Pull request by mmitche**
Pulls in support for additional threading options for xunit

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
     <!-- Input Directories -->
     <PackagesDir Condition="'$(PackagesDir)'==''">$(ProjectDir)packages/</PackagesDir>
-    <ToolsDir Condition="'$(ToolsDir)'==''">$(PackagesDir)Microsoft.DotNet.BuildTools.$(BuildToolsVersion)/lib/</ToolsDir>
+    <dir Condition="'$(ToolsDir)'==''">$(PackagesDir)Microsoft.DotNet.BuildTools.$(BuildToolsVersion)/lib/</ToolsDir>
```
Comment (stephentoub) [inline dir.props]: The start tag was changed to "dir", but the end tag is still "ToolsDir".

Comment (mmitche) [inline dir.props]: Whoops, I think that's the result of typing too fast.  My guess is that was a "dir" into the command prompt.

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
LGTM

**Pull request merged**